library/Rules/MinimumAge.php

```diff
@@ -20,16 +21,16 @@ class MinimumAge extends AbstractRule
 
     public function __construct($age, $format = null)
     {
+        if (!filter_var($age, FILTER_VALIDATE_INT)) {
+            throw new ComponentException('The age must be a numeric value.');
+        }
+
         $this->age = $age;
         $this->format = $format;
     }
 
     public function validate($input)
     {
-        if (!is_int($this->age)) {
```

Comment:
If you just change this line to `filter_var($age, FILTER_VALIDATE_INT)`, and send a pull request agains 0.8 branch, then we can release a path version with this fix, isn't that what you want?